manifests/balancermember.pp

```diff
@@ -40,8 +40,8 @@
 #
 define apache::balancermember (
   String $balancer_cluster,
-  String $url = "http://${$facts['networking']['fqdn']}/",
-  Array $options = [],
+  Stdlib::HTTPUrl $url = "http://${$facts['networking']['fqdn']}/",
+  Array $options       = [],
```

Comment:
Given the other part is a key value, I wonder why options is an array and not a hash but that's probably outside of the scope. The alternative is to make it `Array[Pattern[/=/]]` just to narrow it down to something valid.